### PR TITLE
Removed ViT L, G models from test_encoding

### PR DESCRIPTION
**Pull request by pandu-k**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Test update

* **What is the current behavior?** (You can also link to an open issue here)
very large models means that unit tests don't work on machines with less-than-large amounts of RAM. 

* **What is the new behavior (if this is a feature change)?**
removes the L and G models from unit tests.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Have unit tests been run against this PR?** (Has there also been any additional testing?)
To be done.

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: src/marqo/README.md [hunk 1]
```diff
@@ -23,29 +23,24 @@ git clone https://github.com/marqo-ai/marqo.git
...
-4. `cd` into `src/marqo/tensor_search`
-5. Run the following command:
+4. Run the following command:
 ```bash
 # if you are running Marqo-OS locally: 
-export OPENSEARCH_URL="https://localhost:9200" && 
-    export PYTHONPATH="${PYTHONPATH}:<YOUR ABSOLUTE PATH TO>/marqo/src" &&
-    uvicorn api:app --host 0.0.0.0 --port 8882 --reload
+export OPENSEARCH_URL="https://localhost:9200" 
+export PYTHONPATH="${PYTHONPATH}:$(pwd)/src"
```
Comment (vicilliar) [inline src/marqo/README.md]: user must ensure they are in the marqo directory here, or else $(pwd) will be wrong

File: tests/s2_inference/test_encoding.py [hunk 2]
```diff
@@ -25,9 +25,9 @@ def tearDown(self) -> None:
 
     def test_vectorize(self):
 
-        names = ["multilingual-clip/XLM-Roberta-Large-Vit-B-32",
-                 "onnx16/open_clip/ViT-B-32/laion400m_e32", 'onnx32/open_clip/ViT-B-32-quickgelu/laion400m_e32',
-                 'onnx32/open_clip/ViT-g-14/laion2b_s12b_b42k',"all-MiniLM-L6-v1", "all_datasets_v4_MiniLM-L6", "hf/all-MiniLM-L6-v1", "hf/all_datasets_v4_MiniLM-L6",
+
```
Comment (vicilliar) [inline tests/s2_inference/test_encoding.py]: nit: why the extra new line?

**Review by vicilliar — COMMENTED** [review]
looks good, except for all the added newlines and that the previous commit changes are in this PR for some reason. could we find a way to resolve this?

**Review by vicilliar — CHANGES_REQUESTED** [review]
looks good, except for all the added newlines and that the previous commit changes are in this PR for some reason. could we find a way to resolve this?

File: src/marqo/README.md (hunk 1 above)
Comment (pandu-k) [reply src/marqo/README.md]: This instruction exists above: https://github.com/marqo-ai/marqo/blob/b140f4384c3f3d29dac6c69f20de6ac6561eb109/src/marqo/README.md?plain=1#L23

**Review by wanliAlex — APPROVED** [review]
Looks good!

**Pull request merged**